src/pages/[language]/blog/[slug]/index.vue

```diff
@@ -129,7 +129,10 @@
           <component
             v-if="item.title"
             class="page-blog-post-list__title font-html-blue"
-            :class="headingLevelClassMap[item.headingLevel || defaultHeadingLevel]"
+            :class="`
+              ${headingLevelClassMap[item.headingLevel || defaultHeadingLevel]}
```

Comment:
It feels like this is set up too complicated and now we are complicating it more